### ISSUE TITLE
Bash tab autocomplete not working.

### ISSUE THREAD

**Issue by mjbeyeler** [conda-forge/argcomplete-feedstock#8]
I've proceeded according to this documentation:

https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/enable-tab-completion.html

After installing argcomplete, I have tried with putting `eval "$(register-python-argcomplete conda)"` in `~/.profile`, `~/.bash_profile`, and `~/.bashrc`, and then restarting the terminal to test if conda tab completion works now. Unfortunately, neither of them works.

Do you guys have an idea as to what might be causing the problem? I am using the latest Conda version.

**Comment by asmeurer** [comment]
I think conda removed tab completion support, but they didn't update their documentation.

**Comment by mjbeyeler** [comment]
You're right. They actually mention it in the documentation listed above.

> Conda versions up to 4.3 supports tab completion in bash shells via the argcomplete package. Tab completion is deprecated starting with version 4.4. See issue #415.

Pity.

**Comment by jakirkham** [comment]
Should this be closed then?

**Comment by astier** [comment]
Check out https://github.com/conda/conda/issues/9178 
It says put https://raw.githubusercontent.com/tartansandal/conda/harden-bash-completion/conda/shell/etc/bash_completion.d/conda in `~/.local/share/bash-completion/completions/` this works for me

**Comment by jasonbrudvik** [comment]
Thanks,  that worked perfectly

**Comment by astier** [comment]
There is now a package for this https://github.com/tartansandal/conda-bash-completion with instructions how to install it. I think this should be closed.

**Comment by th0ger** [comment]
> There is now a package for this https://github.com/tartansandal/conda-bash-completion with instructions how to install it. I think this should be closed.

TLDR;
$conda activate base
$conda install conda-bash-completion
<restart bash>
Worked like a charm.

**Comment by lucacerone** [comment]
Hi, installing "conda-bash-completion" in the base environment doesn't work for me, any suggestion on how to debug what might be the issue?

**Comment by tkazik** [comment]
From https://github.com/conda/conda/issues/9178#issuecomment-552297636, try:

```shell
conda install -c tartansandal conda-bash-completion
```
The `conda install conda-bash-completion` wasn't working in my case.
